contents/docs/feature-flags/snippets/groups-flags-go.mdx

```diff
@@ -1,3 +1,19 @@
-[posthog-go](https://posthog.com/docs/integrate/server/go) does not yet support feature flags with groups.
+Using groups with go requires the latest version of [posthog-go](https://posthog.com/docs/integrate/server/go). Update dependencies via:
```

Comment:
Can we specify which version here?

```suggestion
Using groups with Go requires the latest version of [posthog-go](https://posthog.com/docs/integrate/server/go). Update dependencies via:
```

Reply:
somewhat embarrassingly, our Go sdk doesn't have proper versioning in place, so even if users wanted to go to a specific version, they'll do it using the commit hash in github 😬